packages/react/src/components/TopNavigation/top-navigation.scss

```diff
@@ -0,0 +1,58 @@
+.oxygen-top-nav {
```

Comment:
@brionmario get the prefix as a variable

Reply:
Created an issue: https://github.com/wso2/oxygen-ui/issues/10